critiquebrainz/frontend/templates/review/revisions.html

```diff
@@ -42,6 +42,28 @@
 {% block scripts %}
   {{ super() }}
   <script>
+    {% if count > limit %}
+      var current_page = 0;
+      function load_more() {
+        var more_button = $("#more-button");
+        var loading_message = $("#loading-message");
+        more_button.hide();
+        loading_message.show();
+        $.ajax({
+              url: "{{ url_for('review.revisions_more', id=review.id) }}",
+              data: {page: ++current_page}
+            })
+            .done(function (data) {
+              loading_message.hide();
+              $("#results").append(data.results);
+              if (data.more === true) more_button.show();
+            })
+            .fail(function () {
+              alert({{ _('Failed to load older revisions!') | tojson }});
+            });
+      }
+    {% endif %}
```

Comment:
I tried to move the function(with the ```click()``` method on``` $("$more-button"))``` outside of the block for scripts but it stopped working(Maybe because jQuery wasn't getting loaded). It was getting loaded at the end). So, I tried to debug what was the problem with the ```onclick()```. I found [this](http://stackoverflow.com/questions/3371632/why-cant-i-use-onclick-to-execute-a-function-inside-a-jquery-document-ready) (I am new to js and learning js 🙂 ). So, the scope of the js function onclick was searching for should be global. I should have tried the first way in the link.